src/asfvideo.cpp

```diff
@@ -55,6 +55,11 @@ AsfVideo::GUIDTag::GUIDTag(const uint8_t* bytes) {
   memcpy(&data2_, bytes + DWORD, WORD);
   memcpy(&data3_, bytes + DWORD + WORD, WORD);
   std::copy(bytes + QWORD, bytes + 2 * QWORD, data4_.begin());
+  if (isBigEndianPlatform()) {
```

Comment:
In other places where I've seen this pattern, a variable gets set to isBigEndianPlatform() and byteSwap gets called as:

```
    data1_ = byteSwap(data1_, bSwap);
    data2_ = byteSwap(data2_, bSwap);
    data3_ = byteSwap(data3_, bSwap);
```